acq_stat_reports/utils.py

```diff
@@ -0,0 +1,538 @@
+import functools
+import itertools
+from pathlib import Path
+from typing import Any
+
+# from typing import Callable
+import matplotlib.pyplot as plt
+import numpy as np
+
+# import numpy.typing as npt
+from astropy.table import Table
+from chandra_aca.star_probs import binomial_confidence_interval
+
+from acq_stat_reports.config import conf
+
+
+class BinnedData:
```

Comment:
Seems like a nice opportunity to use a dataclass, but at the very least this needs a docstring.

Reply:
I changed it so it is a dataclass, but I'm not sure it adds much in this case.